test/EtherFiRewardsRouter.t.sol

```diff
@@ -54,5 +65,28 @@ contract EtherFiRewardsRouterTest is TestSetup {
         emit EtherFiRewardsRouter.EthSent(address(etherfiRewardsRouterInstance), liquidityPoolAddress, 10 ether);
         etherfiRewardsRouterInstance.withdrawToLiquidityPool();
         vm.stopPrank();
-    }  
+    }
+
+    function test_recoverERC20() public {
+        get_eeth();
+        vm.startPrank(admin);
+        uint256 balanceBefore = eETHInstance.balanceOf(address(treasuryInstance));
+        etherfiRewardsRouterInstance.recoverERC20(address(eETHInstance), 1 ether);
+        uint256 balanceAfter = eETHInstance.balanceOf(address(treasuryInstance));
+        console.log("balanceBefore: ", balanceBefore);
+        console.log("balanceAfter: ", balanceAfter);
+        assertApproxEqAbs(balanceAfter, balanceBefore + 1 ether, 1);
+    }
+
+    function test_setTreasury() public {
+        vm.startPrank(owner);
+        etherfiRewardsRouterInstance.setTreasury(alice);
+        assertEq(etherfiRewardsRouterInstance.treasury(), alice);
+    } 
+
+    function test_setTreasuryFailure() public {
+        vm.startPrank(bob);
+        vm.expectRevert("Ownable: caller is not the owner");
+        etherfiRewardsRouterInstance.setTreasury(alice);
+    } 
 }
```

Comment:
You should configure your editor to newline all your files by default so you don't introduce these git warnings by accident

Reply:
thx